cmake/LibraryDefine.cmake

```diff
@@ -93,7 +93,8 @@ function(OPENEXR_DEFINE_LIBRARY libname)
     string(TOUPPER "${CMAKE_BUILD_TYPE}" uppercase_CMAKE_BUILD_TYPE)
     set(verlibname ${CMAKE_SHARED_LIBRARY_PREFIX}${libname}${OPENEXR_LIB_SUFFIX}${CMAKE_${uppercase_CMAKE_BUILD_TYPE}_POSTFIX}${CMAKE_SHARED_LIBRARY_SUFFIX})
     set(baselibname ${CMAKE_SHARED_LIBRARY_PREFIX}${libname}${CMAKE_${uppercase_CMAKE_BUILD_TYPE}_POSTFIX}${CMAKE_SHARED_LIBRARY_SUFFIX})
-    install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E chdir \"\$ENV\{DESTDIR\}${CMAKE_INSTALL_FULL_LIBDIR}\" ${CMAKE_COMMAND} -E create_symlink ${verlibname} ${baselibname})")
+    file(CREATE_LINK ${verlibname} ${baselibname} SYMBOLIC)
```

Comment:
Does this work right on Windows? I recall that some link thinks don't work quite the same way as you'd expect from Unix.

Reply:
Could someone with a Window please confirm the symlinks are still valid?

Reply:
Based on https://cmake.org/cmake/help/latest/command/file.html#create-link, we could pass `COPY_ON_ERROR` so that cmake will copy instead of symlinking if it fails?

Reply:
Regarding Windows, the entire logic here is inside `if (... AND NOT WIN32)`, so hopefully it's not an issue. And hopefully `cmake -E create_simlink` has a similar effect to `file(CREATE_LINK ... SYMBOLIC)`.

I've added `COPY_ON_ERROR` as a failsafe anyway.